<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>975</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BugKiller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/BKlogo.png</normaloff>:/icon/BKlogo.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>10</y>
     <width>89</width>
     <height>292</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <widget class="QPushButton" name="titleBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>64</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/icon/title.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="welcomeBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>CloudZongYiGBK</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;CloudZongYiGBK&quot;;</string>
      </property>
      <property name="text">
       <string>        欢迎        &gt;</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/hello.png</normaloff>:/icon/hello.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="mainBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>CloudZongYiGBK</family>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;CloudZongYiGBK&quot;;</string>
      </property>
      <property name="text">
       <string>     路况管控      &gt;</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/main.png</normaloff>:/icon/main.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="carMapBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>     车辆控制      &gt;</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/count.png</normaloff>:/icon/count.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="autoNavigationBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>     自动导航     &gt;</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/cut.png</normaloff>:/icon/cut.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="flowCountBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>流量统计  &gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settingBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>        设置        &gt;</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/help.png</normaloff>:/icon/help.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="aboutBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>        关于        &gt;</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/about.png</normaloff>:/icon/about.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="connectBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>87</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>87</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>已断开</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/icon/open.png</normaloff>:/icon/open.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>22</y>
     <width>731</width>
     <height>502</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="mapBoardSkd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(252, 60, 43);</string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="controlBoardSkd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>223</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>223</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
